test/validators/movie.test.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 
-const Joi = require('joi');
-
+const Joi            = require('joi');
```

Comment:
nit: this remove extra spaces here. the reason we don't align the `=` for this case is that we group together imported modules based on where they're coming from; in this case, `Joi` is an external module, whereas `MovieValidator` is a local file that we're importing. See [here](https://github.com/lob/lob-api/blob/master/test/plugins/features/webhooks/controller.test.js#L3-L12) for another example.